libs/ledger-live-common/src/families/filecoin/bridge/account.ts

```diff
@@ -167,15 +167,15 @@ const prepareTransaction = async (a: Account, t: Transaction): Promise<Transacti
 
 const sync = makeSync({ getAccountShape });
 
-const broadcast: BroadcastFnSignature = async ({ signedOperation: { operation, signature } }) => {
+const broadcast: BroadcastFnSignature = async ({ signedOperation }) => {
   // log("debug", "[broadcast] start fn");
-
-  const tx = getTxToBroadcast(operation, signature);
+  const { operation, signature, signatureRaw } = signedOperation;
+  const tx = getTxToBroadcast(operation, signature, signatureRaw as Record<any, unknown>);
```

Comment:
```suggestion
  const tx = getTxToBroadcast(operation, signature, signatureRaw!);
```

Here the issue is that signatureRaw is of type `Record<string, any> | undefined` when the function only allows `Record<string, any>` if we are sure the signatureRaw is not undefined when we call this function it's better to set it explicitely